cassandra/cluster.py

```diff
@@ -2381,6 +2393,14 @@ def add_prepared(self, query_id, prepared_statement):
         with self._prepared_statement_lock:
             self._prepared_statements[query_id] = prepared_statement
 
+    def use_tablets(self, connection, timeout):
+        try:
+            sel_tablets = "SELECT * FROM system.tablets"
```

Comment:
To be precise this query string is a member of `ControlConnection` and it is possible to avoid the duplication by storing this as a member of `Cluster` not  `ControlConnection`, but it will be inconsistent with the place where different `_SELECT_X` query strings are stored. Should I do that anyway?

Reply:
You should be able to refer to it with `ControlConnection._SELECT_TABLETS`

Reply:
Yes, I did that